obj/SalaryCalculator.csproj.nuget.g.targets

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
```

Comment:
please add .gitignore file to remove all the unnecessary files
here's a ling for asp.net core
https://www.toptal.com/developers/gitignore/api/aspnetcore